frontend/src/containers/User/AccountSetting/AccountSetting.test.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { mount } from "enzyme";
+import { Provider } from "react-redux";
+// import { ConnectedRouter } from "connected-react-router";
+// import { Route, Switch } from "react-router-dom";
+
+import AccountSetting from "./AccountSetting";
+import { userActions } from "../../../store/actions/index";
+import { getMockStore } from "../../../test-utils/mocks";
+
+const stubInitialState = {
+    paper: {},
+    auth: {
+        my: {
```

Comment:
This test seems to be also wrong!(`my`) Did you also test your code by really running?